src/NHSD.BuyingCatalogue.Solutions.API/ViewModels/Suppliers/GetSuppliersNameResult.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace NHSD.BuyingCatalogue.Solutions.API.ViewModels.Suppliers
+{
+    public sealed class GetSuppliersNameResult
```

Comment:
The class can be `internal` but in this instance it's probably best to leave it as `public` given that all the other result classes in BAPI are also `public`.

If we decide to change the visibility of the existing classes it will make sense to do all of them together as tech debt.